assets/src/edit-story/components/checklist/constants.js

```diff
@@ -469,6 +475,33 @@ export const PRIORITY_COPY = {
       `${MAX_VIDEO_RESOLUTION}p`
     ),
   },
+  ampValidation: {
+    title: __('Compatibility', 'web-stories'),
+    footer: (
+      <TranslateWithMarkup
+        mapping={{
+          a: (
+            <Link
+              href="https://wordpress.org/plugins/web-stories/#what%20do%20i%20do%20about%20an%20amp%20validation%20issue%20in%20search%20console%3F"
```

Comment:
as far as I can tell, this isn't translatable.

Reply:
It is. See https://br.wordpress.org/plugins/web-stories/ for example.

But I don't think we should link to this page here, honestly.

We should link to https://wp.stories.google/docs/ instead where have the same content.

Reply:
Thank you! This project sometimes makes me acutely aware of my "American" perspective and ignorances :) 

I'll update now.